### ISSUE TITLE
automatic summary missing formatting/html

### ISSUE THREAD

**Issue by bboehmke** [gohugoio/hugo#7926]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.77.0/extended linux/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?
yes

### Description
If a manual summary is used the `.Summary` contains the HTML formatted manual summary but if the default auto summary is used it only contains the raw text (no markdown/no HTML).

In the template I used `{{ .Summary }}` and also tried variations with `plainify` and `safeHTML` as stated in the documentation but with no changes for the automatic summary.

#### Manual

*Example:*
```md
---
title: "manual summary"
date: 2020-10-31T17:00:00+02:00
---

some summary content **with** markdown

<!--more-->

main content
```

*Output:*
```html
<p>some summary content <strong>with</strong> markdown</p>
```

#### Automatic

*Example:*
```md
---
title: "auto summary"
date: 2020-10-31T17:00:00+02:00
---

some summary content **with** markdown

main content
```

*Output:*
```
some summary content with markdown
main content
```

**Comment by arafel** [comment]
Just FYI - confirmed on latest Hugo, macOS arm64.

(I'm really glad you raised this, I thought for sure I was doing something wrong.)

**Comment by jmooring** [comment]
This was resolved with [v0.134.0](https://github.com/gohugoio/hugo/releases/tag/v0.134.0).
